integration/securitypolicy_test.go

```diff
@@ -170,9 +170,13 @@ func TestSecurityPolicyMigration(t *testing.T) {
 
 						newSpec, err := klient.Universal.Portal().Policy().Get(reqCtx, policyOnK8s.Status.PolID)
 						eval.NoErr(err)
-						eval.True(newSpec.MID == previousPolicyID)
+						eval.True(hasSameValues(&policyOnK8s.Spec, newSpec, policyOnK8s.Status.PolID))
```

Comment:
You have added same check below too

Reply:
the one below ensures that we can access the Policy by using the previous ID and it checks that the Policy that is accessed via the previous ID is the same as the one stored in Kubernetes.